decaylanguage/dec/dec.py

```diff
@@ -65,17 +67,18 @@ class DecFileParser(object):
 
     Example
     -------
-    >>> parsed_file = DecFileParser.from_file('my-decay-file.dec')
+    >>> parsed_file = DecFileParser('my-decay-file.dec')
     """
 
     __slots__ = ("_grammar",
                  "_grammar_info",
-                 "_dec_file_name",
+                 "_dec_file_names",
+                 "_dec_file",
                  "_parsed_dec_file",
                  "_parsed_decays",
                  "_include_ccdecays")
 
-    def __init__(self, filename):
+    def __init__(self, filename=None):
```

Comment:
This could be `self, *filenames`

Reply:
The main constructor body would pretty much look just like the current from_files body.

Reply:
Yep, done.